CONTRIBUTING.md

```diff
@@ -0,0 +1,238 @@
+# Code Development Guidelines
+
+Expectations for 2600Hz engineers as well as folks contributing to the platform.
+
+## Coding Standards
+
+The most important rule: **Be consistent both with your code and the project.**
+
+If, in the project, we have been using a variable name to represent an object, continue that in your code. Attempt to mirror existing conventions first, even if you may not agree with them! We can discuss changing it globally but every effort should be made to have a unified and consistent code base.
+
+The full coding standard that we tend to use, and will likely adapt shortly in our documentation effort, is available [here](https://github.com/inaka/erlang_guidelines).
+
+Some of the highlights:
+
+* Loud errors
+  * Don't let unhandled errors and exceptions go unlogged.
+  * If it is a log line that requires manual intervention or action, then it should be documented in "runbook" style for a system operator to act upon.
```

Comment:
@k-anderson can you provide a link to what you mean? Could be a link to where this style exists in the code already, or a web page outlining the philosophy.